src/site/_includes/components/AuthorsDate.js

```diff
@@ -89,12 +89,12 @@ const renderAuthorNames = (pairs) => {
 /**
  * Render an authors card, including any number of authors and an optional date.
  *
- * @param {{authors: !Array<string>, date: ?Date, images: number}} arg
- * @param {Object.<string, Author>} [authorsCollectionArg]
+ * @param {{authors: !Array<string>, date?: Date, images?: number}} arg
+ * @param {Authors} [authorsCollectionArg]
```

Comment:
just checking that `[authorsCollectionArg]` should continue to use square brackets

Reply:
Removed.